release-material/maven-pb/TextileExample/pb/src/test/java/io/textile/pb/ExampleUnitTest.java

```diff
@@ -0,0 +1,17 @@
+package io.textile.pb;
```

Comment:
Basically all the generated java classes need to be copied into here to replace these